verifiers/internal/gcb/verifier.go

```diff
@@ -27,7 +27,9 @@ func GCBVerifierNew() *GCBVerifier {
 // generated by the builderID.
 func (v *GCBVerifier) IsAuthoritativeFor(builderID string) bool {
 	// This verifier only supports the GCB builders.
-	return strings.HasPrefix(builderID, "https://cloudbuild.googleapis.com/GoogleHostedWorker@")
+	// Note: `ParseBuilderID` never fails with `needVersion=false`.
```

Comment:
won't it fail for something like `https://cloudbuild.googleapis.com/GoogleHostedWorker@v2@v3`?

May want to change `strings.Split` to `strings.SplitN` to avoid, but you have a test case that adequately handles this input as an error.

Reply:
mistake from me. You're 100% right. PTAL.